src/main/java/de/tum/in/www1/artemis/security/JiraAuthenticationProvider.java

```diff
@@ -215,11 +223,9 @@ public boolean supports(Class<?> authentication) {
     @Override
     public void addUserToGroup(String username, String group) throws ArtemisAuthenticationException {
         log.info("Add user " + username + " to group " + group + " in JIRA");
-        HttpHeaders headers = HeaderUtil.createAuthorization(JIRA_USER, JIRA_PASSWORD);
         Map<String, Object> body = new HashMap<>();
         body.put("name", username);
-        HttpEntity<?> entity = new HttpEntity<>(body, headers);
-        RestTemplate restTemplate = new RestTemplate();
+        HttpEntity<?> entity = new HttpEntity<>(body);
```

Comment:
For the new usage of this in JIRA, I think we should call `isGroupAvailable` before to throw a better error.
The super admin can enter any group name in the user management, and the group might not exist in JIRA.

Another option (instead of throwing an error) would be to create the group in such a case.